pkg/constants/constants.go

```diff
@@ -303,6 +303,21 @@ const (
 // DefaultModelLocalMountPath is where models will be mounted by the storage-initializer
 const DefaultModelLocalMountPath = "/mnt/models"
 
+// Default path to mount CA bundle secret volume
+const DefaultCaBundleVolumeMountPath = "/etc/ssl/custom-certs"
+
+// Default name for CA bundle file
+const DefaultCaBundleFileName = "cabundle.crt"
+
+// Default secret name that will be created in the user namespace.
+const DefaultGlobalCaBundleConfigMapName = "global-ca-bundle"
+
+// Custom CA Secret Environment Variables
+const (
+	GlobalCaBundleConfigMapNametEnvVarKey   = "CA_BUNDLE_CONFIGMAP_NAME"
+	GlobalCaBundleVolumeMountPathtEnvVarKey = "CA_BUNDLE_VOLUME_MOUNT_POINT"
```

Comment:
Fix typo

```suggestion
	GlobalCaBundleConfigMapNameEnvVarKey   = "CA_BUNDLE_CONFIGMAP_NAME"
	GlobalCaBundleVolumeMountPathEnvVarKey = "CA_BUNDLE_VOLUME_MOUNT_POINT"
```

Nit: the `Global` prefix is somewhat confusing, gives that this applies to both the global bundle, and also the local one.

Reply:
good catch! 
updated
~~~
	CaBundleConfigMapNameEnvVarKey   = "CA_BUNDLE_CONFIGMAP_NAME"
	CaBundleVolumeMountPathEnvVarKey = "CA_BUNDLE_VOLUME_MOUNT_POINT"
~~~